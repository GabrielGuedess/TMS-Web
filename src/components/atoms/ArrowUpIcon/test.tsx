import { render } from '@testing-library/react';

import { ArrowUpIcon } from '.';

describe('<ArrowUpIcon />', () => {
  it('should render the arrow up icon correctly', () => {
    const { container } = render(<ArrowUpIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
