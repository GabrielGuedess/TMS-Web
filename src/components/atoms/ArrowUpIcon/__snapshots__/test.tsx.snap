// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArrowUpIcon /> should render the arrow up icon correctly 1`] = `
<svg
  fill="none"
  height="10"
  viewBox="0 0 10 10"
  width="10"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M4.35716 2.47737L0.908974 5.82987L5.0443e-07 4.94612L5 0.0848689L10 4.94612L9.09103 5.82987L5.64284 2.47737L5.64284 10.0849L4.35716 10.0849L4.35716 2.47737Z"
    fill="currentColor"
  />
</svg>
`;
